src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -432,6 +433,14 @@ public string BackupLocation
             }
         }
 
+        /// <summary>
+        /// Backup files path
+        /// </summary>
+        public int DefaultTimeOut
+        {
+            get { return defaultTimeOut; }
```

Comment:
why do you need a field? Use an auto property if theres a good reason not to.
https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/auto-implemented-properties

Reply:
Please add this const in https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCore/Configuration/Configurations.cs#L14 instead of this class